### ISSUE TITLE
Gradio freezes and kills LoRA training

### ISSUE THREAD

**Issue by koda-dernet** [ace-step/ACE-Step-1.5#188]
EDIT: While this has not fixed the actual bug i am describing, for anyone else going through something similar, a temporary fix is to code a small script (you can ask your favorite LLM for help) to train from a CLI, the training and config scripts are in ./acestep/training and honestly a standalone tensorboard is way better than what you can see in the gradio. Also, specific to RunPod: Use tmux if you want your training to survive. Gradio still seems to be broken and i hope either someone makes a decent frontend or the devs fix the Gradio, since i see many people having problems with it.

**Describe the bug**
After starting the LoRA training in the included Gradio GUI, the process works as expected until a while after saving the nEpoch or a little before. I have set it to save every 100 epochs, after or slightly before this, it continues training on some steps and then gradio freezes for a small ammount of time and covers the training progress, training log and training loss graph with a scary "error" sign; checking on the terminal reveals no error more than the terminal being clearly frozen and not advancing forward, the VRAM is still in usage however, the GPU itself isn't. This is a little more than mildly infuriating since there doesnt seem to be a hardware limitation, i am on an RTX 6000 Ada on RunPod. Afterwards it is possible to start the server again without the need of restarting the pod. Training logs show nothing more than the steps, loss and epoch.

This always seems to happen between step 220 and step 280.

This is really hard to explain further or more technically since there are no errors on terminal nor logs

**To Reproduce**
Start training with these parameters with the Gradio GUI open on RunPod and wait
0.0001 LR
1 batch size
8 gradient accumulation
Save every 100 epochs (currently also 500)
3 shift

**Expected behavior**
A nice experience letting the GPU do its thing while training.

**Screenshots**
I am not sure what else i can put in here other than

<img width="1519" height="545" alt="Image" src="https://github.com/user-attachments/assets/9c1d9308-ef77-4d3b-8c84-bf9a082dd4e7" />

**Desktop:**
 - OS: Linux, running a Jupyter notebook on RunPod
 - Browser Firefox
 - Version 147.0.2

**Additional context**
I am not using an interruptible runtime in runpod, i have also not ran out of storage. 
The models i am using to train are 5Hz 4b and Acestep1.5 base
I am also unsure whether this is a backend or frontend to backend error.
EDIT2: I am now certain it is a frontend to backend error.

**Comment by koda-dernet** [comment]
Well, i made my own standalone training scripts that work both in terminal and has a GUI, so yeah, closed.
